### PR TITLE
Remove disc evidence coversheet

### PR DESCRIPTION
**Pull request by kmahern**
#### What
Remove wicked gem and disc_evidence_coversheet from CCCD codebase

#### Ticket
https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=222&projectKey=CBO&modal=detail&selectedIssue=CBO-1628

#### Why
This code is redundant since Secure File Exchange

#### How
Removed gem and all disc evidence coversheet mvc code.
--------

#### TODO (wip)

 - [ ] item 1
 - [X] item 2

### CODE REVIEW

**Comment by naseberry** [comment]
@kmahern, remember to remove the related assets:

- app/webpack/packs/disc-evidence.js
- app/webpack/stylesheets/pdf.scss

**Comment by kmahern** [comment]
> @kmahern, remember to remove the related assets:
> 
> * app/webpack/packs/disc-evidence.js
> * app/webpack/stylesheets/pdf.scss

👍 Thank you. I have removed these assets.

**Review by jsugarman — CHANGES_REQUESTED** [review]
Great stuff, you can also remove these lines from dockerfile

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docker/Dockerfile#L22

and 

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docker/Dockerfile#L39

**Review by jsugarman — CHANGES_REQUESTED** [review]
and remove this section from docs

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docs/development.md#install-wkhtmltopdf

**Comment by jsugarman** [comment]
and this line

https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/Brewfile#L6

**Review by jrmhaig — APPROVED** [review]
👍 

The merge conflicts are due to the PRs I've just merged but it should be enough just to recreate `.rubocop_todo.yml`.

**Comment by kmahern** [comment]
> Great stuff, you can also remove these lines from dockerfile
> 
> https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docker/Dockerfile#L22
> 
> and
> 
> https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docker/Dockerfile#L39

👍 have removed wkhtmltopdf from dockerfile

**Comment by kmahern** [comment]
> and remove this section from docs
> 
> https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/docs/development.md#install-wkhtmltopdf

👍 Have removed reference to wkhtmltopdf from docs

**Comment by kmahern** [comment]
> and this line
> 
> https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/Brewfile#L6

👍 Have removed wkhtmltopdf from brewfile

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,12 +57,6 @@ references:
         bundle exec rake webpacker:compile
         bundle exec rake jasmine:ci
 
-  _install-wkhtmltopdf: &install-wkhtmltopdf
```
Comment (jsugarman) [inline .circleci/config.yml:60]: 👍 great, forgot about that

**Review by jsugarman — APPROVED** [review]
Looks good 👍

**Pull request merged**